.circleci/config.yml

```diff
@@ -248,19 +248,13 @@ workflows:
               only: /^[0-9]+\.[0-9]+\.[0-9]+/
             branches:
               ignore: /.*/
-  main-branch:
+  in-pr:
     jobs:
       - test_android:
           filters:
             branches:
-              only: main
+                ignore: main
```

Comment:
we also want the tests to be run on `main`, to make sure we catch any merge regression